scio-hdfs/src/main/scala/com/spotify/scio/hdfs/package.scala

```diff
@@ -28,6 +29,7 @@ import com.google.cloud.dataflow.sdk.coders.AvroCoder
 import com.google.cloud.dataflow.sdk.io.hdfs._
 import com.google.cloud.dataflow.sdk.io.{Read, Write}
 import com.google.cloud.dataflow.sdk.options.DataflowPipelineOptions
+import com.google.cloud.dataflow.sdk.runners.DirectPipelineRunner
```

Comment:
Yeah, was checking directly against `DirectPipelineRunner` before switching to use `ScioUtil.isLocalRunner` and this was left in there. Will remove.